### PR TITLE
Refactor dataGen and created PostgreSQL connection and seed

### PR DESCRIPTION
**Pull request by melvinioso**

### CODE REVIEW

File: database/PostgreSQL.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-console */
+const { Sequelize, DataTypes } = require('sequelize');
+
+
+const sequelize = new Sequelize('item_details', 'oso', 'hrr45', {
+  host: 'localhost',
+  dialect: 'postgres',
```
Comment (a-tiller) [inline database/PostgreSQL.js:7]: I didn't realize sequelize would handle postgres as well, that's great!

File: database/dataGen.js [hunk 2]
```diff
@@ -35,7 +35,8 @@ const dataGen = (writer, encoding, callback) => {
         five: fives,
       };
       const numberOfRatings = ones + twos + threes + fours + fives;
-      const starPercentages = JSON.stringify(stars);
+      const starRatings = JSON.stringify(stars);
+      const starPercentages = starRatings.replace(/"/g, "'");
```
Comment (a-tiller) [inline database/dataGen.js:39]: Is this just a quirk of how postgres ingests the data that it dislikes double quotes?

**Review by a-tiller — APPROVED** [review]
Looks good! Just a couple of curiosity questions.

File: database/dataGen.js (hunk 2 above)
Comment (melvinioso) [reply database/dataGen.js:39]: Yeah i had been hearing from other guys that it causes problems when parsing unless done this way

**Pull request merged**